sceptre/cli/drift.py

```diff
@@ -0,0 +1,108 @@
+import click
+from click import Context
+
+from sceptre.context import SceptreContext
+from sceptre.cli.helpers import (
+    catch_exceptions,
+    write
+)
+from sceptre.plan.plan import SceptrePlan
+
+
+BAD_STATUSES = [
+    "DETECTION_FAILED",
+    "TIMED_OUT"
+]
+
+
+@click.group(name="drift")
+def drift_group():
+    """
+    Commands for calling drift detection.
+    """
+    pass
+
+
+@drift_group.command(name="detect", short_help="Run detect stack drift on running stacks.")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def drift_detect(ctx: Context, path: str):
+    """
+    Detect stack drift and return stack drift status.
+
+    In the event that the stack does not exist, we return
+    a DetectionStatus and StackDriftStatus of STACK_DOES_NOT_EXIST.
+
+    In the event that drift detection times out, we return
+    a DetectionStatus and StackDriftStatus of TIMED_OUT.
+
+    The timeout is set at 5 minutes, a value that cannot be configured.
+
+    :param path: The path to execute the command on.
+    :type path: str
```

Comment:
Since these docstrings show up in the `--help` and in the API docs, for these cli function docstrings, we shouldn't have these parameter docs.